src/ProjectNeon/Assets/Features/Effects/StealLife.cs

```diff
@@ -33,7 +34,7 @@ void Execute(Attack attack)
         _effectTarget.Members.ForEach(
             target =>
             {
-                if (target.Equals(attack.Attacker))
+                if (target.Equals(attack.Attacker()))
```

Comment:
Why is this a method instead of a property?

Reply:
@SilasReinagel Fixed